rfc8724.xml

```diff
@@ -186,7 +186,7 @@ Original:
    see this glossary).
 -->
 
-        <dt>Bi:</dt><dd>Bidirectional. Characterizes a Field Descriptor that applies to headers of packets traveling in either direction (Up and Dw, see this glossary).</dd>
+        <dt>Bi:</dt><dd>Bidirectional. Characterizes a Field Descriptor that applies to headers of packets traveling in either direction (Up and Dw).</dd>
```

Comment:
Glossary was confusing. "Terminology section" would have been a better term. But all in all, I think it's better to remove text. When confronted with "Up" and "Dw", the reader will look in the terminology, which is what we want.